api/app/lib/statistics.rb

```diff
@@ -0,0 +1,104 @@
+require "google/apis/analytics_v3"
+require "google/api_client/auth/key_utils"
+
+class Statistics
+  include ActiveModel::Validations
+  include ActiveModel::Conversion
+  extend ActiveModel::Naming
+
+  def id
+    "statistics-set"
+  end
+
+  def initialize(_range = nil)
+    @client = Google::Apis::AnalyticsV3::AnalyticsService.new
+
+    @client.authorization = Signet::OAuth2::Client.new(
```

Comment:
You're using at least one (Signet), perhaps 2 (Google::Apis?) gem that is not declared in our gem file. It works because I have another gem that relies on this gem. If we were to remove that gem, this would break. Please make gem dependencies explicit in the API Gemfile.